core/src/main/java/io/grpc/internal/AbstractManagedChannelImplBuilder.java

```diff
@@ -627,4 +627,11 @@ private T thisT() {
     T thisT = (T) this;
     return thisT;
   }
+
+  /**
+   * Returns the internal offload executor pool for offloading tasks.
+   */
+  protected ObjectPool<? extends Executor> getOffloadExecutorPool() {
```

Comment:
Now that we have this getter can we mark the data member `this.offloadExecutorPool` private and have all the sites (tests) use the getter?